code/src/components/PopularMovieCard.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+
+export const PopularMovieCard = ({id, poster_path, title  ,release_date}) => {
+    
+return (
+  <>
+    <div className="movie-card">
```

Comment:
If your component returns only one root element, there is no need to wrap it around React Fragment.

In this case WE NEED to use React Fragment :

```jsx
return (
  <div>My element 1</div>
  <div>My element 2</div>
);
```

so, after necessary changes it looks like this :

```jsx
return (
	<>
	  <div>My element 1</div>
	  <div>My element 2</div>
	</>
);
```

But in this case WE DON'T NEED to use React Fragment :

```jsx
return (
	<div>My element 1</div>
);
```